server/admin.go

```diff
@@ -923,10 +940,16 @@ func (s *adminServer) ClusterFreeze(
 		// descriptor unconditionally or we won't always be able to unfreeze
 		// (except by restarting a node which holds the first range).
 		if _, err := process(keys.LocalMax, roachpb.KeyMax); err != nil {
-			return nil, err
+			return err
 		}
 	}
-	return &resp, s.waitForStoreFrozen(stores, req.Freeze)
+	if err := stream.Send(&serverpb.ClusterFreezeResponse{
+		RangesAffected: totalAffected,
+		Message:        fmt.Sprintf("proposed freeze to %d ranges", totalAffected),
```

Comment:
Why not change this message to say "unfreeze" or "thaw" instead of "freeze" in the event that `!req.Freeze`?